src/plugins/planout.ts

```diff
@@ -105,21 +105,29 @@ class InterpretedExperiment extends Experiment<Inputs, Params> {
   }
 }
 
-class BaseNamespace extends Namespace.SimpleNamespace<Inputs, Params> {
+export class BaseNamespace extends Namespace.SimpleNamespace<Inputs, Params> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   stack!: { op: string, args: any[] }[];
 
   _defaultExperiment?: InterpretedExperiment;
 
   _defaultExperimentConfig!: PlanoutExperimentConfig;
 
-  constructor(namespace: string) {
+  availableSegments!: number[];
+
+  constructor(namespace: string, numSegments: number = 1000) {
     super({});
     this.setName(namespace);
+
+    // See parent class: https://github.com/rawls238/PlanOut.js/blob/22196f69a3f4964a420295b1de36fc7dd425ea26/es6/namespace.js#L76
+    // Because availableSegments is set up in the constructor, simply overriding numSegments
```

Comment:
The comment is a bit confusing. The code is clear since numSegments is an explicit parameter to the constructor and isn't passed to the super class. I think the more confusing bit is that availableSegments is declared here, but not used anywhere else since. How about instead of this comment, leave one above the availableSegments declaration above? Something like:

```
// Used by the parent class but missing from it's type definition:
// See: https://github.com/rawls238/PlanOut.js/blob/22196f69a3f4964a420295b1de36fc7dd425ea26/es6/namespace.js#L76
```